### PR TITLE
User/xiangweiniu/fix missing tab

### PR DESCRIPTION
**Pull request by niuxw-msft**
#### Related Issues

- Helps #193 

#### Why is this change being made?
The "isApp" switch in SFSClientTool help display has a missing tab. The change is to align it to other switches.

#### What is being changed?
Add one more '\t' in DisplayUsage() of SFSClientTool.

#### How was the change tested?
After build, run the following script to verify that the isApp switch align to other switches.
image

### CODE REVIEW

**Review by JeffMill — APPROVED** [review]

File: samples/tool/SFSClientTool.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@ void DisplayUsage()
         << "Options:" << std::endl
         << "  -h, --help\t\t\tDisplay this help message" << std::endl
         << "  -v, --version\t\t\tDisplay the library version" << std::endl
-        << "  --isApp\t\tIndicates the specific product is an App" << std::endl
+        << "  --isApp\t\t\tIndicates the specific product is an App" << std::endl
```
Comment (JeffMill) [inline samples/tool/SFSClientTool.cpp:33]: tabs always seem iffy to me.  Maybe these \t should be changed to an appropriate number of spaces?

**Review by JeffMill — APPROVED** [review]
approved with comment

File: samples/tool/SFSClientTool.cpp (hunk 1 above)
Comment (arthuraraujo-msft) [reply samples/tool/SFSClientTool.cpp:33]: Hmm, I guess both would work, but that could be another task.

Comment (JeffMill) [reply samples/tool/SFSClientTool.cpp:33]: Seems like overkill to create a new task just to change \t -> spaces, but ultimately your call.

Comment (niuxw-msft) [reply samples/tool/SFSClientTool.cpp:33]: I found std::setw can help format the text. Do you think it's a better way?

    std::cout << std::left << std::setw(30) << "--product <identifier>" << "Name or GUID of the product to be retrieved\n";
    std::cout << std::left << std::setw(30) << "-h, --help" << "Display this help message\n";

Comment (JeffMill) [reply samples/tool/SFSClientTool.cpp:33]: I'm not offended by that suggestion :-)

Comment (niuxw-msft) [reply samples/tool/SFSClientTool.cpp:33]: Ok. I will complete this PR first. And will create another PR to do this change (in both SFSClientTool.cpp IntegrationDOClient.cpp).

Comment (arthuraraujo-msft) [reply samples/tool/SFSClientTool.cpp:33]: I'm not familiar with that. I think spaces should be enough though

**Comment by niuxw-msft** [comment]
close #193

**Pull request merged**